main/webapp/modules/core/externals/suggest/suggest-4_3a.js

```diff
@@ -1364,7 +1364,7 @@
       var correction = response_data.correction;
       if (correction && correction.length) {
         var spell_link = $('<a class="fbs-spell-link" href="#">').text(correction[0])
-          .bind("click.suggest", function(e) {
+          .on("click.suggest", function(e) {
```

Comment:
Those changes are in an external library - are there no prospects at all of upgrading it?